nesta/packages/crunchbase/predict/src/health_keywords.py

```diff
@@ -0,0 +1,13 @@
+health_keywords = [
```

Comment:
store this as a data file, since 

1) It's data 😄 
2) there's a chance that we might want to reproduce this for non-health terms